git_info/cli.py

```diff
@@ -125,6 +126,26 @@ def has_changes(work_dir, debug):
     click.echo(f"::set-output name=has_changes::{'true' if repo.is_dirty() else 'false'}")
 
 
+@run.command(name="last-commit")
+@click.option("--days", default=os.getenv("LAST_COMMIT_DAYS", 10), required=True, type=int)
+@click.option("--debug", is_flag=True)
+def last_commit(debug, days):
+    """
+    Checks remotely if any commit has been pushed in last X days (default: 10)
+    """
+    token = os.getenv('GITHUB_TOKEN')
+    repo_name = os.getenv("GITHUB_REPOSITORY")
+    ref = os.getenv("GITHUB_REF")
+    headers = {"Authorization": f"bearer {token}"}
+    request = requests.get(f"https://api.github.com/repos/{repo_name}/commits?ref={ref}&per_page=1", headers=headers)
+    commit_date = request.json()[0]['commit']['author']['date']
```

Comment:
Worth adding `raise_for_status` i.e. `request.raise_for_status()` after request is made. 

Also I would change `request` variable to just `r`.